src/index.js

```diff
@@ -279,18 +279,33 @@ Consider using the serverless-python-requirements plugin to help you package Pyt
 		}
 	}
 
-	getNodeTracerParameters(token, edgeHost) {
+	getTracerParameters(
+		token,
+		options,
+		equalityToken = ":",
+		trueValue = "true",
+		falseValue = "false"
+	) {
 		if (token === undefined) {
 			throw new this.serverless.classes.Error("Lumigo's tracer token is undefined");
 		}
-		let configuration = [`token: '${token}'`];
-		if (edgeHost) {
-			configuration.push(`edgeHost: '${edgeHost}'`);
+		let configuration = [`token${equalityToken}'${token}'`];
+
+		for (const [key, value] of Object.entries(options)) {
```

Comment:
I'm not a fan of the changes to this method, the logic here is too opaque and I find it hard to understand and be sure that I get it right.

If this is to cater for differences in how you'd configure the Node.js tracer vs Python, then let's just split this into two methods.

Reply:
@theburningmonk but it will be exactly the same function with different parameters, I would you recommend dealing with DRY here?

Reply:
mm... good question, how that I've thought this through, it ain't gonna make much difference :-/

what if we have:
```
getTracerParameters(token, options, equalityToken, trueValue, falseValue) {
  ...
}

getNodeTracerParameters(token, options) {
  return getTracerParameters(token, options, ":", "true", "false");
}

getPythonTracerParameters(token, options) {
  return getTracerParameters(token, options, "=", "True", "False");
}
```

would that be better?

also, a small thing - instead of initializing `configuration` with `token` and then having to filter `token` in the loop, we can just init `configuration` as `[]`.